doc/architecture/database_schemata.md

```diff
@@ -87,8 +87,69 @@ species }|--|| genus : ""
 genus }|--|| subfamily : ""
 subfamily }|--|| family : ""
 
+maps {
+  INT id PK
+  VARCHAR name "NOT NULL"
+  BOOLEAN is_version "NOT NULL"
```

Comment:
Somehow it is always a version, the question is if it is the latest. Do you maybe mean this?

Reply:
I imagined an entry with is_version set to false as the root, similar to the branch head in git. When loading a version of the map, the system would query for all maps with the same root name and is_version set to true. Adopting certain elements of a map version would then be no different than overwriting attributes in/linked to the root map entry.

It may also work with having everything be technically a version, but we would need to be sure that we don't need some kind of a root/base map version.

Reply:
doc/architecture/glossary.md defines version as a snapshot of a map, so everything in maps is actually versions of maps? Maybe we don't need to know what the first/last (root/current) is, as it should be determinable via the date of the version?

Reply:
Yes, probably we should make the versions a separate table?

Reply:
Sounds like a good plan!

Reply:
@NurramoX I ping you so you also know about this conceptual change with to what table the layers should refer.